app/v1/models/requestData/RetrieveForeignPropertyBissRequest.scala

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.models.requestData
+
+case class RetrieveForeignPropertyBissRequest(nino: String, businessId: String, typeOfBusiness: String, taxYear: Option[String])
```

Comment:
typeOfBusiness and taxYear have special types for their values (see uk-property request model for reference)